content/post/building-a-data-practice.md

```diff
@@ -0,0 +1,85 @@
+
+### Plan your roadmap
+One of the perennial Data team struggles is balancing the short term and the long term. You will be busy right away, pulled in multiple directions, but you should take the time to form opinions about the future. How would you like to evolve data tooling, infrastructure, processes, and people?
+
+Think about what is missing, what can be improved, and how they align with the organization's priorities. Are you using Google Analytics for tracking? That might be fine for now, but many businesses will eventually need something more powerful. How are reports being run? If data is primarily accessed and explored in spreadsheets, you should think about options for BI tools. Who should be hired next? A data analyst? A data engineer?
+
+Form opinions about your roadmap now so you will be prepared to advocate for it whenever it comes up. What will you say when someone in marketing decides they want to integrate a new service, or when an engineer starts to build a data tool in house, or when the founders raise more funding and ask you how to spend it?
+
+### Create a data dictionary
+As you learn things, especially things that are surprising, write them down. Think about what the next data hire will need to know. Avoid the urge to make this fancy — a Google doc is fine.
+
+The concept of a data dictionary is a bit controversial. What do you include? What don't you include? How do you maintain it? For more mature organizations, I recommend considering [Carl Anderson's suggestions](https://medium.com/@leapingllamas/data-dictionary-a-how-to-and-best-practices-a09a685dcd61) for using a data dictionary to create a shared understanding around key metrics and vocabulary.
+
+In this case, though, a data dictionary is primarily for you and future data hires. Even though the business and its data will change over time, there are still key pieces of information that are useful to document. Are timezones in UTC or your organization's time? What are some core pieces of business logic? How are key metrics calculated? Are there important database tables or column names that are not self-explanatory?
+
+At Kickstarter, we called this document *Common Data Gotchas*, which is a good way to think about it.
+
+### Use a project management tool
+Start tracking all the work you do, even quick data requests. Write down the question, the results, and how you got them. Encourage others in your organization to interact with you in this tool. Documenting your work in this way will be important for reproducibility, it will be good process as your data team grows, and it will help with visibility and transparency, demonstrating your impact on the business.
+
+My first choice for this is Trello, but use whatever integrates well into your organization.
+
+### Create communication channels for data
```

Comment:
Another consideration:

These few paragraphs are related to the topic of setting stakeholder expectations. I've found that there are two classes of coworkers (here anyway):

1. Those who've worked with Data teams and have a certain expectation/vision.
2. Those who haven't and aren't sure what to expect.

You'll have to temper the expectations of the former, e.g. "Looker won't be up and running tomorrow and here are the reasons why". I think the expectations of the latter group are well addressed in this post.

Reply:
Another great point. You can definitely tell the kinds of groups I've interacted with the most. :)

I think going deeper on stakeholder management is a little outside the scope here, but that could make an interesting post in the future.